### PR TITLE
Ft/day 8

### PR DESCRIPTION
**Pull request by Samael19IT5**
Hw day 7

### CODE REVIEW

File: app/controllers/order_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  STANDARD = 2
+  PREMIUM = 5
+  VIP = 10
+
+  def total_price(order)
+    details = OrderDetail.where('order_id=' + order.id)
+    sum = 0
+    details.each { |detail|
+      book = Book.where('id=' + detail.book_id.to_s)
```
Comment (ttduongtran) [inline app/controllers/order_controller.rb]: Did you test this function? 

Where will return array[]
so here if you want to use book.price, you need to use `book.first`

File: app/controllers/order_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      sum += book.price * detail.quantity
+    }
+    sum
+  end
+
+  def price(order)
+    user = User.where('id=' + order.user_id)
+    if user.level == 0
```
Comment (ttduongtran) [inline app/controllers/order_controller.rb:19]: Don't if in if. 
To easy, you can write: 

```
if user.level == 0
      discount = STANDARD
    elsif user.level == 1
        discount = PREMIUM
else
        discount = VIP
    end

File: app/models/post.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 class Post < ApplicationRecord
   belongs_to :category
+  default_scope -> { where(status: true).order(created_at: :desc) }
```
Comment (ttduongtran) [inline app/models/post.rb:3]: Nice!

**Review by ttduongtran — CHANGES_REQUESTED** [review]

File: app/controllers/order_controller.rb (hunk 1 above)
Comment (Samael19IT5) [reply app/controllers/order_controller.rb]: i'll check it again.

File: app/controllers/order_controller.rb (hunk 2 above)
Comment (Samael19IT5) [reply app/controllers/order_controller.rb:19]: i got it

**Pull request closed without merging**